client_test.go

```diff
@@ -35,10 +37,22 @@ func TestNewClient(t *testing.T) {
 
 	conf := cl.(*marathonClient).config
 
-	assert.Equal(t, conf.HTTPClient, http.DefaultClient)
+	assert.Equal(t, conf.HTTPClient, defaultHttpClient)
+	assert.Equal(t, conf.HTTPSSEClient, defaultHttpSSEClient)
```

Comment:
Let's also assert here that the timeout on our default SSE client is not set to protect ourselves.